wurst/extensions/ItemPoolExtensions.wurst

```diff
@@ -0,0 +1,7 @@
+package ItemPoolExtensions
+
+public function itempool.addItemType(int itemId, real weight)
+    ItemPoolAddItemType(this, itemId, weight)
+
+public function itempool.getRandomItem(vec2 pos) returns item
```

Comment:
I think this makes more sense as `placeRandomItem`.

Reply:
@jonathanmclaus 👍